doc/generators.md

```diff
@@ -42,10 +42,19 @@ For directed graphs, allocates an $n \times n$ sparse matrix of boolean as an ad
 random_configuration_model(n::Int, k::Array{Int}; seed=-1)
 ```
 
-Creates a random undirected graph according to the [configuraton model](http://tuvalu.santafe.edu/~aaronc/courses/5352/fall2013/csci5352_2013_L11.pdf). It contains `n` vertices, the vertex `ì` having degree `k[i]`.
+Creates a random undirected graph according to the [configuraton model](http://tuvalu.santafe.edu/~aaronc/courses/5352/fall2013/csci5352_2013_L11.pdf). It contains `n` vertices, the vertex `i` having degree `k[i]`.
 
 Defining `c = mean(k)`, it allocates an array of `nc` `Int`s, and takes approximately $nc^2$ time.
 
+Make sure that `k` is a graphical sequence (see `isgraphical`), otherwise the function won't ever return.
```

Comment:
It can be an optional precondition (defaulting to `false`, if you'd like, though I think `true` is safer) as well as a separate check (so keep your code.